### DISCUSSION TITLE
sndio support

### DISCUSSION THREAD

**Discussion by demostanis** [polybar/polybar#2753] [Ideas]
This is the default OpenBSD sound server, and it has been ported to Linux and other BSDs.
Is supporting it planned?
https://sndio.org/

**Comment by patrick96** [comment]
No support is planned. I'm sure something like [pulseaudio-tail](https://github.com/polybar/polybar-scripts/tree/master/polybar-scripts/pulseaudio-tail) can be implemented using a custom script module for sndio.

But there will not be a native polybar module for sndio.
